topi/python/topi/x86/__init__.py

```diff
@@ -10,3 +10,4 @@
 from .pooling import schedule_pool, schedule_global_pool
 from .bitserial_conv2d import schedule_bitserial_conv2d
 from .depthwise_conv2d import schedule_depthwise_conv2d_NCHWc
+from .dense import _schedule_dense, _schedule_dense_pack, _schedule_dense_nopack
```

Comment:
Could you also add `from .batch_matmul import schedule_batch_matmul` in the init as well?

Reply:
done, waiting on tests.